### PR TITLE
dev-php/fedora-autoloader: Add version bump 1.0.0

### PR DESCRIPTION
**Pull request by GuillaumeSeren**
Closes: https://bugs.gentoo.org/639764
Package-Manager: Portage-2.3.16, Repoman-2.3.6

### CODE REVIEW

File: dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+EAPI=6
+
+DESCRIPTION="Fedora's Static PSR-4, PSR-0, and classmap autoloader"
+HOMEPAGE="https://github.com/php-fedora/autoloader"
+SRC_URI="https://github.com/php-fedora/autoloader/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (orlitzky) [inline dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild]: The ebuild for fedora-autoloader-0.2.1 now has

```
KEYWORDS="alpha amd64 arm hppa ia64 ppc ppc64 ~s390 ~sh sparc x86"
```

Can you please add all of those to the 1.0.0 ebuild (as ~arch)?

Comment (GuillaumeSeren) [reply dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild]: Yes I added it back 👍

**Comment by GuillaumeSeren** [comment]
As explained on the Gentoo BTS, the test are restricted because the need new packages like >=friendsofphp/php-cs-fixer-2.0 I will work on it after this one

File: dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild (hunk 1 above)
Comment (orlitzky) [reply dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild]: Can you make the ebuild unstable on all arches pleeeaase? Repoman will complain about adding a new version that is already stable:

```
RepoMan scours the neighborhood...
  KEYWORDS.stable [fatal]       1
   dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild added with stable keywords: alpha amd64 arm hppa ia64 ppc ppc64 sparc x86
```

Comment (GuillaumeSeren) [reply dev-php/fedora-autoloader/fedora-autoloader-1.0.0.ebuild]: @orlitzky Yes this is why I removed them in the first place but I should have changed them to unstable, it is now done 🍺

**Comment by orlitzky** [comment]
There was one last conflict with the `Manifest` file because we've changed our hashes recently. To avoid another round trip, I ran `repoman manifest` and committed the new hash under your name. I hope you're OK with that.

**Comment by GuillaumeSeren** [comment]
Great ! thank you @orlitzky 😄

**Pull request closed without merging**